src/gbxml/MapEnvelope.cpp

```diff
@@ -194,8 +219,29 @@ namespace gbxml {
       material.setSpecificHeat(specificHeat);
       material.setRoughness(roughness);
 
-    } else if (!rvalueElement.empty())  //Material no mass that has only R-value
-    {
+      if (extir && intir) {
+        material.setThermalAbsorptance((*extir + *intir) / 2.0);  // average of the two?
```

Comment:
@joseph-robertson typically I `extir` would be impact the outside material of a layered construction, and `intir` would impact the inner most material. I don't believe that no mass material can be exposed inside or outside material layer in E+. But another single layer construction like concrete, would be problematic if it had different inside and outside absorptance values.

Reply:
@joseph-robertson if a construction has two or more material layers then for the exposed materials one should get outside value and the other inside. But if you have a construction with single material layer, or if the same material is used on inside and outside then you can use average. It doesn't matter much for materials like insulation that are not exposed.

Reply:
@DavidGoldwasser So are you implying, because the existing FT behavior doesn't consider where the Material layer is relative to the construction, that the FT is incorrect?

Could you provide me with a concrete example for mapping between OSM Material and GBXML Material?